scipy/ndimage/morphology.py

```diff
@@ -506,8 +506,8 @@ def binary_dilation(input, structure=None, iterations=1, mask=None,
                            output, border_value, origin, 1, brute_force)
 
 
-def binary_opening(input, structure=None, iterations=1, output=None,
-                   origin=0):
+def binary_opening(input, structure=None, iterations=1, mask=None,
+        output=None, border_value=0, origin=0, brute_force=False):
```

Comment:
You can only add new keywords at the end, because of backwards compatibility. If a user is now calling this function with all positional arguments, their code should remain working.